install-dev/upgrade/upgrade.php

```diff
@@ -45,8 +45,11 @@
     define('_PS_ADMIN_DIR_', base64_decode($_GET['adminDir']));
 }
 
-require_once dirname(__FILE__).'/../init.php';
+require_once __DIR__ . '/../../vendor/autoload.php';
+require_once __DIR__ . '/../../config/defines.inc.php';
+require_once __DIR__ . '/../../config/autoload.php';
```

Comment:
This is already done in `init.php` I think

Reply:
Yes but `Upgrade::migrateSettingsFile()` needs to be called before the init so that the kernel is properly instanciated And I guess this method has dependency on const which are defined in these other files

Reply:
Precisely, thanks @jolelievre